### PR TITLE
Php70 scoped

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: syntax_error_note_php70.md [hunk 1]
```diff
@@ -1,29 +1,13 @@
...
-    171|      * Tries to create new style instance from string.
-    172|      */
-  > 173|     private function createStyleFromString(string $string) : ?\RectorPrefix20210516\_HumbugBox0b2f2d5c77b8\Symfony\Component\Console\Formatter\OutputFormatterStyleInterface
-    174|     {
-    175|         if (isset($this->styles[$string])) {
-Syntax error in rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Formatter/OutputFormatter.php on line 173
+Parse error: rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Event/ConsoleCommandEvent.php:25
+    23|      * The return code for skipped commands, this will also be passed into the terminate event.
+    24|      */
+  > 25|     public const RETURN_CODE_DISABLED = 113;
```
Comment (TomasVotruba) [inline syntax_error_note_php70.md]: Maybe consider increating the print limit to 40 seconds.
There is much more files to print

Comment (samsonasik) [reply syntax_error_note_php70.md]: ok, increased to 40 https://github.com/rectorphp/rector-src/pull/50/commits/e9f4601252af7b4f7654df85d65a37da1cb28118

**Comment by samsonasik** [comment]
@TomasVotruba finally 🎉 1 error to go which is a symplify `ParameterSkipper`'s const that has `@noRector` that fixed in PR https://github.com/symplify/symplify/pull/3202 . 

Need https://github.com/symplify/symplify/pull/3202 to be review/merged and new release of `symplify/autowire-array-parameter`

**Comment by samsonasik** [comment]
Close - Re-open to restart build

**Comment by samsonasik** [comment]
@TomasVotruba it works 🎉 🎉 🎉 https://github.com/rectorphp/rector-prefixed-php70/commit/b6cd1797c8b4e7c2e160110b10d53f863be92ac8

**Comment by samsonasik** [comment]
I think https://github.com/rectorphp/rector-prefixed-php70 need to be cleanup first to ensure no left over files like old .github actions workflows that still exists.

**Comment by samsonasik** [comment]
merging to `php70` for now

**Pull request merged**